README.md

```diff
@@ -103,8 +103,8 @@ Please note for this to work you should create/update `~/.gradle/gradle.properti
     signing.password=
     signing.secretKeyRingFile=
 
-### Publishing the jars without signing to a local repository ###
-    ./gradlew -Dorg.gradle.project.skipSigning=true -Dorg.gradle.project.mavenUrl=file://path/to/repo uploadArchivesAll
+### Installing the jars to the local Maven repository ###
```

Comment:
If I recall correctly, one of the reasons I included this example with `uploadArchivesAll` was because `./gradlew install` doesn't generate all the artifacts. So if you're trying to make sure everything works and you'll be able to do a release but don't want to do a real upload of the resulting archives, you need to do what I previously had here.

It's possible nobody else cared much about this and I did since I did Confluent's packages for Kafka and ran into this then, but I've also used this as part of my sanity check on build script changes to make sure everything still works. Not sure whether it's important enough toe keep given the documentation of options this patch is adding.